src/components/nav-bar/nav-bar.jsx

```diff
@@ -0,0 +1,38 @@
+import react from 'react';
```

Comment:
You have a typo here, importing 'react' is a wrong way. see how you have done it in other components